charts/rode-collector-harbor/values.yaml

```diff
@@ -0,0 +1,87 @@
+# Default values for rode-collector-sonarqube.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: ghcr.io/rode/collector-harbor
+  pullPolicy: IfNotPresent
+  tag: ""
```

Comment:
should we set a default tag?

Reply:
The deployment template uses the .Chart.AppVersion if image.tag is not set